tests/functional/features/application_restricted.feature

```diff
@@ -124,7 +124,7 @@ Feature: Unattended Access
     And I create a new app
     And I add the attribute with key of apim-app-flow-vars and a value of { "pds" : { "app-restricted": { "update": true } }}
     And I add the scope urn:nhsd:apim:app:level3:personal-demographics-service
-    And I wait for 200 milliseconds
+    And I wait for 500 milliseconds
```

Comment:
Increased wait times to reduce likelihood of an intermittent. The additional 300/400 ms on each test will be made up by not having to rerun a 5+ minute stage.